dashboard/app/models/levels/star_wars_grid.rb

```diff
@@ -0,0 +1,81 @@
+# == Schema Information
+#
+# Table name: levels
+#
+#  id                       :integer          not null, primary key
+#  game_id                  :integer
+#  name                     :string(255)      not null
+#  created_at               :datetime
+#  updated_at               :datetime
+#  level_num                :string(255)
+#  ideal_level_source_id    :integer
+#  solution_level_source_id :integer
+#  user_id                  :integer
+#  properties               :text(65535)
+#  type                     :string(255)
+#  md5                      :string(255)
+#  published                :boolean          default(FALSE), not null
+#  notes                    :text(65535)
+#  audit_log                :text(65535)
+#
+# Indexes
+#
+#  index_levels_on_game_id  (game_id)
+#  index_levels_on_name     (name)
+#
+
+class StarWarsGrid < Studio
+  def self.create_from_level_builder(params, level_params)
+    level = new(
+      level_params.merge(
+        user: params[:user],
+        game: Game.custom_studio,
+        level_num: 'custom',
+        start_blocks: '<xml><block type="when_run"/></xml>',
+        wall_map_collisions: true,
+        block_moving_into_walls: true,
```

Comment:
These options will go away with @balderdash's change to enable these whenever a collision map is present.